src/user/login.cjsx

```diff
@@ -1,65 +1,34 @@
 React = require 'react'
+classnames = require 'classnames'
 
-api   = require '../api'
+AccountsIframe = require './accounts-iframe-mixin'
 User  = require './model'
 
 UserLogin = React.createClass
 
+  mixins: [AccountsIframe]
```

Comment:
React is moving away from mixins. Could this be redone without them? (Maybe a separate PR?)